tests/global-fetch.spec.ts

```diff
@@ -266,16 +266,23 @@ it('should remove content-length from reidrected post requests', async ({ playwr
 });
 
 
-const serialization = [
+const serialization: [string, any][] = [
   ['object', { 'foo': 'bar' }],
   ['array', ['foo', 'bar', 2021]],
   ['string', 'foo'],
+  ['string (falsey)', ''],
   ['bool', true],
+  ['bool (false)', false],
   ['number', 2021],
+  ['number (falsey)', 0],
+  ['null', null],
+  ['literal string null', 'null'],
+  ['literal string undefined', 'undefined'],
 ];
 for (const [type, value] of serialization) {
   const stringifiedValue = JSON.stringify(value);
   it(`should json stringify ${type} body when content-type is application/json`, async ({ playwright, server }) => {
+    it.fail(type === 'literal string null', `Some where isn't escaping this. Perhaps either at the following link or before: https://github.com/microsoft/playwright/blob/bfe7b7cc20ab9fdbf63dc84986369bf0f3f5a2dd/packages/playwright-core/src/protocol/serializers.ts#L53`);
```

Comment:
@dgozman This is a new test case since the last review. It remains unfixed. `'null'` (the string) gets treated by the channel serializer (https://github.com/microsoft/playwright/blob/bfe7b7cc20ab9fdbf63dc84986369bf0f3f5a2dd/packages/playwright-core/src/protocol/serializers.ts#L53) extra specially, so we'll need to do something different somewhere. I didn't dig into this particular failure enough to determine where the appropriate fix for this new case would be. Let's tackle it in a follow up Issue and PR? If you know the correct fix off the top of your head, feel free to do patch it post merge, otherwise let me know and I can dig in.

Reply:
I think [this function](https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/fetch.ts#L550-L562) parses `'null'` as `null` and converts from a string to a null. I've discussed this with Yury and we decided to ignore this case, so let's remove it from the test :) We can leave the comment in the `isJsonParsable` function though.